### PR TITLE
MANTA-5195 switch to 'uuid' package for UUID generation

### PR DESCRIPTION
**Pull request by trentm**

### CODE REVIEW

**Comment by trentm** [comment]
ZOMG, got the test suite ot pass against the UFDS in my COAL:

```
[14:41:39 trentm@bluesteel:~/joy/node-ufds (MANTA-5195 n:0.10)]
$ UFDS_LDAP_ROOT_PASSWORD=d63e56ce8187ff070769574964a3f264 UFDS_IP=10.192.0.18 make test
npm install
./node_modules/.bin/nodeunit test/*.test.js --reporter tap
TAP version 13
# testBasicConnect
connect
ok 1 (unnamed assert)
# testBadCreds
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:51.723Z","v":0}
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":50,"bindDN":"cn=root","err":{"message":"cn=root","name":"InvalidCredentialsError","stack":"InvalidCredentialsError: cn=root\n    at messageCallback (/Users/trentm/joy/node-ufds/node_modules/ldapjs/lib/client/client.js:1217:40)\n    at Parser.onMessage (/Users/trentm/joy/node-ufds/node_modules/ldapjs/lib/client/client.js:833:14)\n    at Parser.emit (events.js:95:17)\n    at Parser.write (/Users/trentm/joy/node-ufds/node_modules/ldapjs/lib/messages/parser.js:105:8)\n    at CleartextStream.onData (/Users/trentm/joy/node-ufds/node_modules/ldapjs/lib/client/client.js:820:26)\n    at CleartextStream.emit (events.js:95:17)\n    at CleartextStream.<anonymous> (_stream_readable.js:765:14)\n    at CleartextStream.emit (events.js:92:17)\n    at emitReadable_ (_stream_readable.js:427:10)\n    at _stream_readable.js:420:7","code":49},"msg":"UFDS: invalid credentials; aborting","time":"2020-04-17T21:41:51.864Z","v":0}
ok 2 (unnamed assert)
ok 3 (unnamed assert)
# testQueryVersion
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:52.031Z","v":0}
ok 4 ufds version v17
# testGetUser
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:53.384Z","v":0}
ok 5 (unnamed assert)
ok 6 (unnamed assert)
ok 7 (unnamed assert)
ok 8 (unnamed assert)
ok 9 (unnamed assert)
ok 10 has dclocalconfig property
ok 11 dclocalconfig is empty
# testGetUserByUuid
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:53.751Z","v":0}
ok 12 (unnamed assert)
ok 13 (unnamed assert)
# testGetUserByEmail
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:54.045Z","v":0}
ok 14 (unnamed assert)
ok 15 (unnamed assert)
# testGetUserNotFound
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:54.225Z","v":0}
ok 16 (unnamed assert)
ok 17 (unnamed assert)
ok 18 (unnamed assert)
ok 19 (unnamed assert)
ok 20 (unnamed assert)
# testGetUserExByUuid
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:54.505Z","v":0}
ok 21 (unnamed assert)
ok 22 (unnamed assert)
# testGetUserExByLogin
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:54.763Z","v":0}
ok 23 (unnamed assert)
ok 24 (unnamed assert)
# testGetUserExByLoginNotFound
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:54.928Z","v":0}
ok 25 (unnamed assert)
ok 26 (unnamed assert)
ok 27 (unnamed assert)
ok 28 (unnamed assert)
ok 29 (unnamed assert)
# testGetUserExByUuidNotFound
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:55.102Z","v":0}
ok 30 (unnamed assert)
ok 31 (unnamed assert)
ok 32 (unnamed assert)
ok 33 (unnamed assert)
ok 34 (unnamed assert)
# testGetUserExNoCrossOver0
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:55.284Z","v":0}
ok 35 (unnamed assert)
ok 36 (unnamed assert)
ok 37 (unnamed assert)
ok 38 (unnamed assert)
ok 39 (unnamed assert)
# testGetUserExNoCrossOver1
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:55.449Z","v":0}
ok 40 (unnamed assert)
ok 41 (unnamed assert)
ok 42 (unnamed assert)
ok 43 (unnamed assert)
ok 44 (unnamed assert)
# testGetUserExError0
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:55.585Z","v":0}
ok 45 (unnamed assert)
# testGetUserExError1
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:55.714Z","v":0}
ok 46 (unnamed assert)
# testGetUserExError2
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:55.844Z","v":0}
ok 47 (unnamed assert)
# testEmptyListDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:55.991Z","v":0}
ok 48 (unnamed assert)
ok 49 null dclocalconfig
# testAddDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:56.182Z","v":0}
ok 50 (unnamed assert)
ok 51 added cfg
ok 52 dn correct
ok 53 dclocalconfig correct
ok 54 defaultfabricsetup correct
# testGetUserWithDcConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:56.396Z","v":0}
ok 55 (unnamed assert)
ok 56 user
ok 57 has dc config
ok 58 dn correct
ok 59 dclocalconfig correct
ok 60 defaultfabricsetup correct
# testGetDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:56.544Z","v":0}
ok 61 (unnamed assert)
ok 62 found config object
ok 63 dn correct
ok 64 dclocalconfig correct
ok 65 defaultfabricsetup correct
# testListDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:56.705Z","v":0}
ok 66 (unnamed assert)
ok 67 found config object
ok 68 dn correct
ok 69 dclocalconfig correct
ok 70 defaultfabricsetup correct
# testUpdateDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:56.918Z","v":0}
ok 71 (unnamed assert)
ok 72 updated config object
ok 73 dn correct
ok 74 dclocalconfig correct
ok 75 defaultfabricsetup updated
ok 76 defaultnetwork updated
# testDelUpdateDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:57.122Z","v":0}
ok 77 (unnamed assert)
ok 78 config object
ok 79 dclocalconfig still present
ok 80 correctly deleted
# testDeleteDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:57.319Z","v":0}
ok 81 (unnamed assert)
ok 82 err
ok 83 cfg object not found
# testAuthenticate
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:57.690Z","v":0}
ok 84 (unnamed assert)
ok 85 (unnamed assert)
ok 86 (unnamed assert)
ok 87 (unnamed assert)
# testAuthenticateByUuid
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:57.979Z","v":0}
ok 88 (unnamed assert)
ok 89 (unnamed assert)
ok 90 (unnamed assert)
ok 91 (unnamed assert)
# testAddKey
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:58.381Z","v":0}
ok 92 (unnamed assert)
ok 93 (unnamed assert)
ok 94 have key: [object Object]
ok 95 (unnamed assert)
ok 96 (unnamed assert)
ok 97 (unnamed assert)
# testAddDuplicatedKeyNotAllowed
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:58.914Z","v":0}
ok 98 (unnamed assert)
ok 99 add duplicated key error
# testListAndGetKeys
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:59.264Z","v":0}
ok 100 (unnamed assert)
ok 101 (unnamed assert)
ok 102 (unnamed assert)
ok 103 (unnamed assert)
ok 104 (unnamed assert)
ok 105 (unnamed assert)
ok 106 (unnamed assert)
ok 107 (unnamed assert)
ok 108 (unnamed assert)
# testAddKeyByName
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:41:59.910Z","v":0}
ok 109 (unnamed assert)
ok 110 (unnamed assert)
ok 111 (unnamed assert)
ok 112 (unnamed assert)
ok 113 (unnamed assert)
# testAddDuplicatedKeyByName
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:00.892Z","v":0}
ok 114 (unnamed assert)
ok 115 add duplicated key error
# testDelKey
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:01.307Z","v":0}
ok 116 (unnamed assert)
ok 117 (unnamed assert)
ok 118 (unnamed assert)
ok 119 (unnamed assert)
# testUserGroups
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:01.822Z","v":0}
ok 120 (unnamed assert)
ok 121 (unnamed assert)
ok 122 (unnamed assert)
ok 123 (unnamed assert)
ok 124 (unnamed assert)
ok 125 (unnamed assert)
ok 126 (unnamed assert)
ok 127 (unnamed assert)
ok 128 (unnamed assert)
ok 129 (unnamed assert)
ok 130 (unnamed assert)
ok 131 (unnamed assert)
# testCrudUser
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:02.213Z","v":0}
ok 132 (unnamed assert)
ok 133 (unnamed assert)
ok 134 (unnamed assert)
ok 135 (unnamed assert)
ok 136 (unnamed assert)
ok 137 (unnamed assert)
ok 138 (unnamed assert)
ok 139 (unnamed assert)
ok 140 (unnamed assert)
# testCrudLimit
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:02.739Z","v":0}
ok 141 (unnamed assert)
ok 142 (unnamed assert)
ok 143 (unnamed assert)
ok 144 (unnamed assert)
ok 145 (unnamed assert)
ok 146 (unnamed assert)
ok 147 (unnamed assert)
ok 148 (unnamed assert)
ok 149 (unnamed assert)
ok 150 (unnamed assert)
ok 151 (unnamed assert)
ok 152 (unnamed assert)
ok 153 (unnamed assert)
ok 154 (unnamed assert)
ok 155 (unnamed assert)
# testMetadata
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:03.718Z","v":0}
ok 156 (unnamed assert)
ok 157 (unnamed assert)
ok 158 (unnamed assert)
ok 159 (unnamed assert)
ok 160 (unnamed assert)
ok 161 (unnamed assert)
ok 162 (unnamed assert)
ok 163 (unnamed assert)
ok 164 (unnamed assert)
ok 165 (unnamed assert)
ok 166 (unnamed assert)
ok 167 (unnamed assert)
ok 168 (unnamed assert)
ok 169 (unnamed assert)
# testAddSubUserToAccount
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:04.109Z","v":0}
ok 170 (unnamed assert)
ok 171 returned new user
ok 172 login correct
ok 173 uuid set
ok 174 SUB_UUID was already set
ok 175 (unnamed assert)
ok 176 sub_login correct
# testGetUserExSubUserByUuid
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:04.288Z","v":0}
ok 177 (unnamed assert)
ok 178 expected subuser login
# testGetUserExSubUserByLogin
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:04.458Z","v":0}
ok 179 (unnamed assert)
ok 180 expected subuser login
# testGetUserExSubUserWrongLogin
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:04.593Z","v":0}
ok 181 error expected
ok 182 expected statusCode
ok 183 expected restCode
ok 184 an error message was set
ok 185 no user object expected
# testGetUserExSubUserWrongAccount
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:04.725Z","v":0}
ok 186 error expected
ok 187 expected statusCode
ok 188 expected restCode
ok 189 an error message was set
ok 190 no user object expected
# testAddSubUserDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:04.894Z","v":0}
ok 191 (unnamed assert)
ok 192 added cfg
ok 193 dn correct
ok 194 defaultfabricsetup correct
# getSubUserWithDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:05.078Z","v":0}
ok 195 (unnamed assert)
ok 196 user
ok 197 has dc config
ok 198 dn correct
ok 199 dclocalconfig correct
# delSubUserDcLocalConfig
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:05.257Z","v":0}
ok 200 (unnamed assert)
ok 201 expected err
ok 202 cfg object not found
# testSubuserKey
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:05.574Z","v":0}
ok 203 (unnamed assert)
ok 204 (unnamed assert)
ok 205 have key: [object Object]
ok 206 (unnamed assert)
ok 207 (unnamed assert)
ok 208 (unnamed assert)
ok 209 (unnamed assert)
ok 210 (unnamed assert)
ok 211 (unnamed assert)
ok 212 (unnamed assert)
ok 213 (unnamed assert)
ok 214 (unnamed assert)
# testSubUsersMetadata
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:05.943Z","v":0}
ok 215 (unnamed assert)
ok 216 metadata user
ok 217 (unnamed assert)
ok 218 metadata cn
ok 219 metadata cn value
ok 220 metadata dn
ok 221 metadata dn value
ok 222 meta objectclass
ok 223 meta objectclass val
ok 224 (unnamed assert)
ok 225 get meta w/o object
ok 226 (unnamed assert)
ok 227 get meta with object
ok 228 (unnamed assert)
# testSubUsersLimits
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:06.284Z","v":0}
ok 229 (unnamed assert)
ok 230 (unnamed assert)
ok 231 (unnamed assert)
ok 232 (unnamed assert)
ok 233 (unnamed assert)
ok 234 (unnamed assert)
ok 235 subuser
ok 236 (unnamed assert)
ok 237 (unnamed assert)
ok 238 (unnamed assert)
ok 239 (unnamed assert)
ok 240 (unnamed assert)
ok 241 (unnamed assert)
ok 242 (unnamed assert)
ok 243 (unnamed assert)
# testSubUsersCrud
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:06.758Z","v":0}
ok 244 (unnamed assert)
ok 245 (unnamed assert)
ok 246 (unnamed assert)
ok 247 (unnamed assert)
ok 248 (unnamed assert)
ok 249 (unnamed assert)
ok 250 (unnamed assert)
ok 251 (unnamed assert)
ok 252 (unnamed assert)
ok 253 (unnamed assert)
# testSubUsersCrudWithObject
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:07.090Z","v":0}
ok 254 (unnamed assert)
ok 255 correct user returned
ok 256 user has "account" set
ok 257 (unnamed assert)
ok 258 (unnamed assert)
ok 259 phone updated
ok 260 (unnamed assert)
# testSubUsersCrudWithObjectMismatchedAccount
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:07.346Z","v":0}
ok 261 (unnamed assert)
ok 262 correct user returned
ok 263 user has "account" set
ok 264 (unnamed assert)
ok 265 (unnamed assert)
# testAccountPolicies
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:07.732Z","v":0}
ok 266 (unnamed assert)
ok 267 (unnamed assert)
ok 268 (unnamed assert)
ok 269 Array of policies
ok 270 (unnamed assert)
ok 271 (unnamed assert)
ok 272 (unnamed assert)
ok 273 (unnamed assert)
# testAccountRoles
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:08.129Z","v":0}
ok 274 (unnamed assert)
ok 275 (unnamed assert)
ok 276 (unnamed assert)
ok 277 Array of roles
ok 278 (unnamed assert)
ok 279 (unnamed assert)
ok 280 subuser
ok 281 (unnamed assert)
ok 282 user roles is an array
ok 283 (unnamed assert)
ok 284 sub user default roles is an array
ok 285 (unnamed assert)
ok 286 (unnamed assert)
ok 287 (unnamed assert)
# testAccountDisabled
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:08.256Z","v":0}
ok 288 skipped for UFDS < v19
# testRemoveUserFromAccount
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:08.445Z","v":0}
ok 289 (unnamed assert)
# testHiddenControl
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:08.826Z","v":0}
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:08.829Z","v":0}
ok 290 (unnamed assert)
ok 291 (unnamed assert)
ok 292 (unnamed assert)
ok 293 (unnamed assert)
# testAccountResources
{"name":"ufds_unit_test","hostname":"bluesteel.local","pid":59484,"component":"ufds","level":30,"msg":"LDAP client disconnected","time":"2020-04-17T21:42:09.078Z","v":0}
ok 294 (unnamed assert)
ok 295 (unnamed assert)
ok 296 (unnamed assert)
ok 297 Array of resources
ok 298 (unnamed assert)
ok 299 (unnamed assert)
ok 300 (unnamed assert)
ok 301 (unnamed assert)

1..301
# tests 301
# pass  301

# ok
```

File: test/ufds.test.js [hunk 1]
```diff
@@ -22,7 +20,12 @@ var UFDS = require('../lib/index');
 
 // --- Globals
 
-var UFDS_URL = process.env.UFDS_URL || 'ldaps://10.99.99.18';
+assert.string(process.env.UFDS_IP, 'UFDS_IP envvar');
+assert.string(process.env.UFDS_LDAP_ROOT_PASSWORD,
+    'UFDS_LDAP_ROOT_PASSWORD envvar');
+
+var UFDS_URL = 'ldaps://' + process.env.UFDS_IP;
+var UFDS_PASSWORD = process.env.UFDS_LDAP_ROOT_PASSWORD
```
Comment (twhiteman) [inline test/ufds.test.js:28]: Same again.

File: test/connect.test.js [hunk 2]
```diff
@@ -5,30 +5,36 @@
...
 // --- Globals
 
-var UFDS_URL = 'ldaps://' + (process.env.UFDS_IP || '10.99.99.18');
+assert.string(process.env.UFDS_IP, 'UFDS_IP envvar');
+assert.string(process.env.UFDS_LDAP_ROOT_PASSWORD,
+    'UFDS_LDAP_ROOT_PASSWORD envvar');
 
-var LOG = new Logger({
+var UFDS_URL = 'ldaps://' + process.env.UFDS_IP;
+var UFDS_PASSWORD = process.env.UFDS_LDAP_ROOT_PASSWORD
```
Comment (twhiteman) [inline test/connect.test.js:25]: This is missing a semicolon ... which then breaks `make check`.

File: test/ufds.test.js (hunk 1 above)
Comment (trentm) [reply test/ufds.test.js:28]: Sorry about that. See https://github.com/joyent/node-ufds/pull/6

File: test/connect.test.js (hunk 2 above)
Comment (trentm) [reply test/connect.test.js:25]: Sorry about that. See https://github.com/joyent/node-ufds/pull/6

**Pull request merged**